ui/tract-renderer/src/main/java/org/cru/godtools/tract/activity/TractActivity.kt

```diff
@@ -38,7 +38,9 @@ import org.cru.godtools.base.Settings.Companion.FEATURE_TUTORIAL_LIVE_SHARE
 import org.cru.godtools.base.URI_SHARE_BASE
 import org.cru.godtools.base.model.Event
 import org.cru.godtools.base.tool.EXTRA_SHOW_TIPS
+import org.cru.godtools.base.tool.SHORTCUT_LAUNCH
 import org.cru.godtools.base.tool.activity.BaseToolActivity
+import org.cru.godtools.base.tool.analytics.model.ToolOpenedViaShortcutAnalyticsActionEvent
```

Comment:
ktlint will complain about these unused imports. there is a code auto-format task within Android Studio that should deal with the majority of ktlint violations